sous-chef/tests/unit_tests/recipe_book/test_read_recipe_book.py

```diff
@@ -519,6 +519,60 @@ def test__select_random_recipe_weighted_by_rating_removes_long_cooking_time(
             ).squeeze(),
         )
 
+    @staticmethod
+    @pytest.mark.parametrize(
+        "method,item_type",
+        [
+            ("get_random_recipe_by_category", "categories"),
+            ("get_random_recipe_by_tag", "tags"),
+        ],
+    )
+    @pytest.mark.parametrize(
+        "recipe_title,recipe_rating",
+        [
+            ("Excluded by rating", 2),
+            ("Not excluded by rating", 5),
+        ],
+    )
+    def test__select_random_recipe_weighted_by_rating_removes_low_rating(
```

Comment:
True that what you wrote does this too but a bit confused since the commit is for excluding unrated recipes and the test is showing it for values above a certain amount

Reply:
yes, tried to keep the unit test case and core logic a bit more general. But that's why I was wondering if we needed an integration test specific to the unrated business.